web_external/ImagesGallery/Facets/ImagesFacetView.js

```diff
@@ -14,15 +14,29 @@ isic.views.ImagesFacetView = isic.View.extend({
         this.filter = settings.filter;
 
         this.facetId = this.completeFacet.id;
+        this.facetContentId = this.className + '-' + this.completeFacet.id.replace(/\./g, '-');
```

Comment:
A minor nit, but let's set this using `this.facetId` instead of `this.completeFacet.id`.